examples/serve_incoming.rs

```diff
@@ -0,0 +1,16 @@
+#![deny(warnings)]
```

Comment:
I'd suggest removing the example, it doesn't show too much, and having too many can make it harder to know what examples to look at to learn.

Reply:
Removed.